arm/Microsoft.RecoveryServices/vaults/protectionContainers/readme.md

```diff
@@ -1,19 +1,19 @@
-# RecoveryServicesProtectionContainer `[Microsoft.RecoveryServices/vaults/protectionContainers]`
+# RecoveryServices Vaults ProtectionContainers `[Microsoft.RecoveryServices/vaults/protectionContainers]`
 
-This module deploys a Protection Container for a Recovery Services Vault
+This module deploys RecoveryServices Vaults ProtectionContainers.
+// TODO: Replace Resource and fill in description
```

Comment:
Did you delete and recreate the file? It now has again several placeholders and autogenerated descriptions